indigo_app/static/javascript/indigo/traditions.js

```diff
@@ -133,8 +133,8 @@
         },
       },
     },
-    // default annotation style
-    annotationStyle: 'annotationGeneratorBase',
+    // default remark style
+    remarkStyle: 'remarkGeneratorBase',
```

Comment:
Perhaps `remarkGenerator` is better than `remarkStyle`? The generate determines the style, but this is a reference to the generator.